src/Login.js

```diff
@@ -18,14 +18,28 @@ class Login extends Component {
     this.setState({ [event.target.name]: event.target.value })
   }
 
+  clearInputs() {
+    this.setState({
+      email: '',
+      password: '',
+      error: ''
+    })
+  }
+
   handleLogin = (event) => {
     event.preventDefault();
     getUserData(this.state.email, this.state.password)
     .then(user => this.props.setUser(user))
-    .catch(error => alert("NOPE"))
+    .catch(error => this.setState({ error: error }))
```

Comment:
Same concern as above.. is this message really going to be what we want the user to see?

Reply:
Let's test this by enacting the `.catch()` and logging the `error` to confirm the data type of the error.